components/search/src/filter.rs

```diff
@@ -338,84 +342,61 @@ mod tests {
     static ENGINES_LIST: Lazy<Vec<SearchEngineDefinition>> = Lazy::new(|| {
         vec![
             SearchEngineDefinition {
-                aliases: Vec::new(),
-                charset: String::from("UTF-8"),
-                classification: SearchEngineClassification::General,
                 identifier: "engine1".to_string(),
-                partner_code: String::new(),
                 name: "Test".to_string(),
-                order_hint: None,
-                telemetry_suffix: None,
                 urls: SearchEngineUrls {
                     search: SearchEngineUrl {
                         base: "https://example.com".to_string(),
                         method: "GET".to_string(),
                         params: Vec::new(),
                         search_term_param_name: None,
                     },
-                    suggestions: None,
-                    trending: None,
+                    ..Default::default()
                 },
+                ..Default::default()
             },
             SearchEngineDefinition {
-                aliases: Vec::new(),
-                charset: "UTF-8".to_string(),
-                classification: SearchEngineClassification::General,
                 identifier: "engine2".to_string(),
                 partner_code: String::new(),
```

Comment:
I noticed partnerCode isn't assigned a specific value. Could partner code be removed here?